clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoCommandLine.java

```diff
@@ -476,6 +476,16 @@ protected void handleUserCommand() {
 
     Command.setAuthenticationMode(auth, userName);
 
+    if (CommandActions.LIST.equals(command)) {
+      newListUsers(url, ignore, metalake).handle();
+      return;
+    }
+
+    if (Objects.isNull(user)) {
+      System.err.println("User is undefined. Please use --user to specify a user.");
```

Comment:
I'd keep in line with other error messages and use "Missing --user option." even if we were to use more verbose error messages I'd avoid saying "undefined".